internal/mapper/resource_mapper.go

```diff
@@ -171,18 +171,21 @@ func generateResourceSchema(logger *slog.Logger, explorerResource explorer.Resou
 				continue
 			}
 
-			parameterAttribute, schemaErr := s.BuildResourceAttribute(paramName, schema.ComputedOptional)
+			paramPath := strings.Split(paramName, ".")
+
+			parameterAttribute, schemaErr := s.BuildResourceAttribute(paramPath[len(paramPath)-1], schema.ComputedOptional)
 			if schemaErr != nil {
 				log.WarnLogOnError(pLogger, schemaErr, "skipping mapping of read operation parameter")
 				continue
 			}
 
-			readParameterAttributes = append(readParameterAttributes, parameterAttribute)
+			// TODO: currently, no errors can be returned from merging, but in the future we should consider raising errors/warnings for unexpected scenarios, like type mismatches between attribute schemas
+			readResponseAttributes, _ = readResponseAttributes.MergeAttribute(paramPath, parameterAttribute, schema.ComputedOptional)
 		}
 	}
 
 	// TODO: currently, no errors can be returned from merging, but in the future we should consider raising errors/warnings for unexpected scenarios, like type mismatches between attribute schemas
-	resourceAttributes, _ := createRequestAttributes.Merge(createResponseAttributes, readResponseAttributes, readParameterAttributes)
+	resourceAttributes, _ := createRequestAttributes.Merge(createResponseAttributes, readResponseAttributes)
```

Comment:
With the introduction of "common" parameters in #114, it's possible a parameter could be intended for a nested attribute from the create response/request as well.

I think to avoid problems with that, we can move this final merge to happen **before** the parameters are merged, i.e. before line 143. And then update line 183 to merge with `resourceAttributes`.

So the order would be:
- Merge `createRequestAttributes` -> `createResponseAttributes` -> `readResponseAttributes`. Resulting in `resourceAttributes`
- Then merge each individual parameter attribute into `resourceAttributes`